agenda-services/src/main/java/org/exoplatform/agenda/util/NotificationUtils.java

```diff
@@ -755,7 +769,7 @@ private static final void setSpaceName(NotificationInfo notification, TemplateCo
     if (identity == null) {
       templateContext.put(TEMPLATE_VARIABLE_AGENDA_NAME, "");
     } else {
-      String spaceName = spaceService.getSpaceByPrettyName(identity.getRemoteId()).getDisplayName();
+      String spaceName = spaceService.getSpaceByPrettyName(identity.getRemoteId())==null ? null : spaceService.getSpaceByPrettyName(identity.getRemoteId()).getDisplayName();
```

Comment:
```suggestion
      Space space = spaceService.getSpaceByPrettyName(identity.getRemoteId());
      String spaceName = space == null ? null : space.getDisplayName();
```
Calling twice `spaceService.getSpaceByPrettyName` isn't performant in fact